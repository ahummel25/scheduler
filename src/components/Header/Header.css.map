{"version":3,"sources":["input"],"names":[],"mappings":"AAAA;EACE,2BAAA;EACA,aAAA;EACA,YAAA;;AAHF,OAKE;EACE,gBAAA;EACA,eAAA;;AAIJ;EACE,wCAAA;EACA,YAAA;;AAGF;EACE;IACE,WAAW,YAAX;;EAEF;IACE,WAAW,cAAX"}