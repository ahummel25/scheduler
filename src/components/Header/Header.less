.header {
  background-color: slateblue;
  height: 120px;
  color: white;

  .header-title {
    font-size: 1.5em;
    margin-top: 0px;
  }
}

.logo {
  animation: logo-spin infinite 20s linear;
  height: 80px;
}

@keyframes logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}