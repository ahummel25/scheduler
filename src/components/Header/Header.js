import React from 'react';
import logo from '../../logo.svg';
import './Header.css';

class Header extends React.Component {
  render() {
    return (
      <div>
        <header className="header">
          <img src={logo} className="logo" alt="logo" />
          <h1 className="header-title">Scheduler</h1>
        </header>
      </div>
    );
  }
}

export default Header;