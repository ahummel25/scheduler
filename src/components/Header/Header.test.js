import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

test('should render Header correctly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper.find('h1').length).toBe(1);
  expect(wrapper.find('h1').text()).toEqual('Scheduler')
  expect(wrapper).toMatchSnapshot();
});

