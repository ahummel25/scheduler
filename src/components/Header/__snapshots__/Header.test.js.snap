// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <header
    className="header"
  >
    <img
      alt="logo"
      className="logo"
      src="logo.svg"
    />
    <h1
      className="header-title"
    >
      Scheduler
    </h1>
  </header>
</div>
`;
